profiles/api/oauth2.py

```diff
@@ -118,21 +118,20 @@ def _token() -> Response:
 
         json_data = json.loads(response.text)
 
-        if 'access_token' not in json_data:
-            raise ValueError('No access_token')
-        elif 'expires_in' not in json_data:
-            raise ValueError('No expires_in')
-        elif json_data.get('token_type').lower() != 'bearer':
-            raise ValueError('Got token_type {}, expected Bearer'.format(
-                json_data.get('token_type')))
-
         json_data = {key: json_data[key] for key in
```

Comment:
doesn't this raise `KeyError` when the field is not there?

Reply:
Yep, which is fine (ORCID have given us a broken response). The `ValueError`s above were redundant.